TSC-Charter.md

```diff
@@ -74,9 +74,32 @@ including:
 * Development process and any coding standards.
 * Mediating technical conflicts between Collaborators or Foundation
   projects.
+* Establishing and stewarding a consensus-seeking process to represent the technical community and its work in external communications.
 
 The TSC will define Node.js project’s release vehicles.
 
+The communication oversight responsibilities include:
+
+* Technical development processes of the project
+* Releases and release announcements
+* Technical documentation
+* Contributor onboarding and conduct
+* Project governance
+* Technical direction
+
+Specific exclusions from the communication oversight responsibilities include:
+
+* Brand marketing
+* Fundraising and financial sponsorship (including, but not limited to, the OpenJS Foundation's financial sponsorship of the project)
+* Legal matters
+* Marketing/ community events
+
+Communication on excluded oversight areas that intersect with included responsibilities will be handled in collaboration with the OpenJS Foundation.
+
+The community may establish project-specific communication channels, such as social media accounts and discussion forums, subject to the above requirements. If the communication channels are used to represent the project, administrator access should be granted to both the TSC and the Foundation for oversight, while the day-to-day operation can be delegated to another party as long as they respect the consensus seeking process of the project.
```

Comment:
Trying to be more specific again since the idea of "the Foundation" is vague..

```suggestion
The community may establish project-specific communication channels, such as social media accounts and discussion forums, subject to the above requirements. If the communication channels are used to represent the project, administrator access should be granted to both the TSC and staff delegated by the Foundation for oversight, while the day-to-day operation can be delegated to another party as long as they respect the consensus seeking process of the project.
```